__tests__/project.test.js

```diff
@@ -0,0 +1,39 @@
+const db = require("../db/connection.js");
+const { app } = require("../db/seeds/app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const test_data = require("../db/data/test-data"); // auto looks for index.js
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(test_data); //< seed(seed)
+});
+
+describe("using the App.js to run our database of topics", () => {
+  describe("GET request along the path of /api/topics.. returns an array of topics", () => {
+    test(" get an array of topics with the status code 200", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          expect(Array.isArray(response.body.topics)).toBe(true);
```

Comment:
technically we don't need this line as line 25 infers that it is an array (forEach only works on arrays)